tests/rpm/pkg_rpm_basic_test.py

```diff
@@ -74,6 +74,7 @@ def test_scriptlet_content(self):
 
     def test_basic_headers(self):
         common_fields = {
+            "EPOCH": b"1",
```

Comment:
Can you pass in the check for EPOCH to only the samples which add it?
I know that is not the same pattern here, but what is here was hacked up in a hurry. Having version and release the same for everything is not so bad, but we should have split out noarch and group so they can vary by test case.

Reply:
Done